docs/content/config/advanced/ios-mail-push-support.md

```diff
@@ -0,0 +1,214 @@
+- You currently cannot use the same account UUID on multiple devices. This means that if you use the same backup on multiple devices (e.g. old phone / new phone) only one of them will get the notification. Use different backups or recreate the mail account.
+
+## Privacy concerns
+
+- The service does not send any part of the actual message to Apple.
+- The information sent contains the device UUID to notify and the (on-device) account UUID which was generated by the iOS mail application when creating the account.
+- Upon receiving the notification, the iOS mail application will connect to the IMAP server given by the provided account UUID and fetch the mail to notify the user.
+- Apple therefore does not know the mail address for which the mail was received, only that a specific account on a specific device should be notified that a new mail or that a mail was moved to the INBOX folder.
+
+## Installation
+
+Both components will be built using Docker and included into a custom `docker-mailserver` image. Afterwards the required configuration is added to `docker-data/dms/config`. The registration data is stored in `/var/mail-state/lib-xapsd` to be consistent with `ONE_DIR=1`.
+
+1. Create a Dockerfile to build a `docker-mailserver` image that includes the [dovecot-xaps-plugin](https://github.com/freswa/dovecot-xaps-plugin) as well as the [dovecot-xaps-daemon](https://github.com/freswa/dovecot-xaps-daemon). This is required to ensure that the Dovecot plugin is built against the same Dovecot version. The `:edge` tag is used here, but you might want to use a released version instead.
+
+    ```txt
+    FROM mailserver/docker-mailserver:edge AS dovecot-plugin-xaps
+    RUN apt-get update && \
+        apt-get -y --no-install-recommends install git cmake make build-essential dovecot-dev && \
+        mkdir /tmp/dovecot-xaps-plugin && \
+        cd /tmp/dovecot-xaps-plugin && \
+        git clone https://github.com/freswa/dovecot-xaps-plugin.git . && \
+        mkdir build && \
+        cd build && \
+        cmake .. -DCMAKE_BUILD_TYPE=Release && \
+        make install
+
+    FROM golang:1.19-alpine AS dovecot-xaps-daemon
+    ENV GOPROXY=https://proxy.golang.org,direct
+    ENV CGO_ENABLED=0
+    WORKDIR /go
+    RUN apk add --no-cache --virtual build-dependencies git && \
+        mkdir /go/dovecot-xaps-daemon && \
+        cd /go/dovecot-xaps-daemon && \
+        git clone https://github.com/freswa/dovecot-xaps-daemon . && \
+        go build ./cmd/xapsd
+
+    FROM mailserver/docker-mailserver:edge
+    COPY --from=dovecot-plugin-xaps /usr/lib/dovecot/modules/*_xaps_* /usr/lib/dovecot/modules/
+    COPY --from=dovecot-xaps-daemon /go/dovecot-xaps-daemon/xapsd /usr/bin/xapsd
+
+    # create a non-root user for the daemon process as well as configuration and run state directories
+    RUN adduser --quiet --system --group --disabled-password --home /var/mail-state/lib-xapsd --no-create-home xapsd && \
+        mkdir -p /var/run/xapsd /etc/xapsd
```

Comment:
Yeah. I thought about using a specific uid, but I was afraid that it might get used later down the road. So I opted for the safe approach and perform a `chown`.